automation/src/test/java/com/microsoft/identity/common/test/automation/utility/Constants.java

```diff
@@ -0,0 +1,36 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.common.test.automation.utility;
+
+public class Constants {
+
+    public static final String OUTLOOK_CLIENT_ID = "27922004-5251-4030-b22d-91ecd9a37ea4";
+
+    public static final String ONE_DRIVE_CLIENT_ID = "b26aadf8-566f-4478-926f-589f601d9c74";
+
+    public static final String OUTLOOK_REDIRECT_URI = "urn:ietf:wg:oauth:2.0:oob";
```

Comment:
You may just want to create constants for this well known redirect URI rather than for both one drive and outlook.